test/api/createJobTests.js

```diff
@@ -0,0 +1,39 @@
+const proxyquire = require('proxyquire');
+const sinon = require("sinon");
+const chai = require("chai");
+const assert = chai.assert;
+
+const auth = {
+    url: "test",
+    user: "user",
+    password: "password"
+};
+
+const jobArgs = {
+        repository: "test",
+        branch: "master",
+        name: "test"
+    }
+
+const dataMock = {name:"Test"};
+
+const JenkinsWrapper = {
+    job: {
+        create: sinon.stub().yields(null, dataMock)
+    }
+};
+
+const jenkinsStub = () =>
+    JenkinsWrapper;
```

Comment:
It would look nicer to have all in the same line `const jenkinsStub = () => JenkinsWrapper;`